### ISSUE TITLE
[Bug][A11Y][Phone Number] Set default screen-reader focus on selected country in selector

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-flutter#496]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

.

### OS Version

Android and iOS

### App version

alpha 0.6.0 (509)

### Describe the issue

With Voice Over and Talkback, when I open the country selector, the focus is set on Urugay on iOS and Turkmenistan on Android (instead of UK by default)

### Expected Behavior

Focus should be set on current selected country

**Comment by AhmedAmineZr** [comment]
@pya35 we can't make focus on selected item on dropdown menu in flutter, is not native behavior
